src/app/containers/Ad/Amp/index.jsx

```diff
@@ -19,6 +20,12 @@ import { ServiceContext } from '#contexts/ServiceContext';
 
 const FullWidthWrapper = styled.div`
   background-color: ${C_LUNAR_LIGHT};
+
+  .amp-geo-pending &,
```

Comment:
Setting this visibility here means that we will see content shifting when the ads load in. There is a discussion on the ads paper document to decide on how we want to approach this.